src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -336,6 +363,27 @@ void EnzymesSelectorWidget::sl_filterTextChanged(const QString& filterText) {
     }
 }
 
+void EnzymesSelectorWidget::sl_findSingleEnzymeTaskStateChanged() {
+    auto t = qobject_cast<FindSingleEnzymeTask*>(sender());
+    SAFE_POINT(t != nullptr, L10N::nullPointerError("FindSingleEnzymeTask"), );
```

Comment:
No need to localize safe points: their errors are not visible to users.

Reply:
But I do not do any localization, do I?

Reply:
Same

Reply:
OK